README.md

```diff
@@ -115,51 +115,49 @@ You can specify all arguments above multiple times.
 ### Model Manipulation
 
 Sometimes, your model will output different results than what you expect.
-For example, `メロンパン` is split into two tokens in the following command.
+For example, `外国人参政権` is split into wrong tokens in the following command.
 We use `--scores` option to show the score of each character boundary:
 ```
-% echo '朝食はメロンパン1個だった' | cargo run --release -p predict -- --scores --model path/to/jp-0.4.7-5-tokenize.model.zst
-朝食 は メロン パン 1 個 だっ た
-0:朝食 -15398
-1:食は 24623
-2:はメ 30261
-3:メロ -26885
-4:ロン -38896
-5:ンパ 8162
-6:パン -23416
-7:ン１ 23513
-8:１個 18435
-9:個だ 24964
-10:だっ -15065
-11:った 14178
+% echo '外国人参政権と政権交代' | cargo run --release -p predict -- --scores --model path/to/jp-0.4.7-5-tokenize.model.zst
+外国 人 参 政権 と 政権 交代
+0:外国 -11785
+1:国人 16634
+2:人参 5450
+3:参政 4480
+4:政権 -3697
+5:権と 17702
+6:と政 18699
+7:政権 -12742
+8:権交 14578
+9:交代 -7658
 ```
 
-To concatenate `メロンパン` into a single token, manipulate the model in the following steps so that the score of `ンパ` becomes negative:
+The correct is `外国 人 参政 権`.
+To split `外国人参政権` into correct tokens, manipulate the model in the following steps so that the score of `参政権` becomes inverted:
```

Comment:
```suggestion
To split `外国人参政権` into correct tokens, manipulate the model in the following steps so that the sign of score of `参政権` becomes inverted:
```